etc/kayobe/inspector.yml

```diff
@@ -30,17 +30,31 @@
 #inspector_manage_firewall:
 
 # List of of default inspector processing plugins.
-#inspector_processing_hooks_default:
+inspector_processing_hooks_default:
+  - ramdisk_error
+  - validate_interfaces
+  - capabilities
+  - pci_devices
+  - local_link_connection
+  - lldp_basic
 
 # List of of additional inspector processing plugins.
 #inspector_processing_hooks_extra:
+inspector_processing_hooks_extra:
+  # Adds switch system name to ironic port local_link_connection.
+  - system_name_llc
+  # Assigns physical_network to ironic port based on switch system name.
+  - system_name_physnet
+  # Processes additional inventory data collected by the extra-hardware
+  # collector.
+  - extra_hardware
 
 # List of of additional inspector processing plugins.
 #inspector_processing_hooks:
 
 # Which MAC addresses to add as ports during introspection. One of 'all',
 # 'active' or 'pxe'.
-#inspector_add_ports:
+inspector_add_ports: pxe
```

Comment:
It can be useful to add all ports if you have a multi-rail box. Could be done separately but would require reinspection.

Reply:
Cool, we will be doing this in future I think, for now we have the nodes PXE booting on a single 25GbE interface and then using that for all user traffic.